recipe/migrations/osx_arm64.txt

```diff
@@ -1067,3 +1067,6 @@ pybamm
 nifty
 arosics
 sensormapgeo
+modin
+xopen
+keras
```

Comment:
keras is noarch, so doesn't need to be migrated. Can you remove?

Reply:
They would still need to have it if its dependencies need migration

It is also a good idea in the event it ceases to be `noarch` (then we get a migration PR in short order)

Reply:
Ok, fair enough. First time I've seen a noarch package put in the migration.

Reply:
Right it will be a no-op for the `noarch` package. Though if any of its current (or future) dependencies are not `noarch`, they will get migrated.